src/providers/user/user-provider.js

```diff
@@ -92,6 +95,29 @@ export default class UserProvider extends EventEmitter {
         });
     }
 
+    async canSetMissionStatus() {
+        const user = await this.getCurrentUser();
+        const writeParameters = user.getWriteParameters();
+
+        const areParametersStatus = await Promise.all(
+            writeParameters.map(parameterName => isMissionStatusParameter(parameterName))
+        );
+
+        return areParametersStatus.some(isParameterStatus => isParameterStatus);
+    }
+
+    async getPossibleMissionRoles() {
+        const possibleStatuses = await this.missionStatus.getAllMissionRoles();
+
+        return possibleStatuses;
+    }
+
+    async getPossibleMissionRoleStatuses() {
+        const statusRoles = await this.missionStatus.getPossibleMissionStatuses();
```

Comment:
Change this variable name to `possibleMissionStatuses` since we're getting mission statuses.
Perhaps confused with the `getPossibleMissionRoles` above?